README.md

```diff
@@ -351,7 +351,7 @@ webhooks.on(eventNames, handler)
     </td>
     <td>
       <strong>Required.</strong>
-      Name of the event. One of <a href="#listofalleventnames">GitHub’s supported event names</a>.
+      Name of the event. One of <a target="_blank" rel="noopener noreferrer" href="https://developer.github.com/webhooks/#events">GitHub’s supported event names</a>.
```

Comment:
Can you leave out the `target` and `rel` properties? GitHub doesn’t add these either when it transpiles Markdown links to HTML

Reply:
Might as well use Markdown syntax here instead

Reply:
No worries, fixed 👍